Cargo.toml

```diff
@@ -54,8 +54,8 @@ xxhash-rust = { version = "0.8.6", features = ["xxh3"] }
 
 [workspace.dependencies.arrow]
 package = "arrow2"
-git = "https://github.com/jorgecarleitao/arrow2"
-rev = "ba6a882bc1542b0b899774b696ebea77482b5c31"
+git = "https://github.com/reswqa/arrow2"
+branch = "to_large_list"
```

Comment:
I will change the arrow2 dependency to the correct upstream rev after that being merged.

Reply:
Merged upstream. Can you rebease?

Reply:
Sure, rebased and updated to new rev.